qiskit/backends/local/__init__.py

```diff
@@ -14,3 +14,6 @@
 from .statevector_simulator_cpp import StatevectorSimulatorCpp
 from .statevector_simulator_py import StatevectorSimulatorPy
 from .unitary_simulator_py import UnitarySimulatorPy
+
+# Global instance to be used as the entry point for convenience.
+Aer = LocalProvider()  # pylint: disable=invalid-name
```

Comment:
Why are we adding Aer bits here? I feel like we should do that separately when it's time to really move to Aer instead of the local simulators